docs/advanced_usages/as-python-lib.md

```diff
@@ -81,7 +84,10 @@ if __name__ == "__main__":
 ??? note "How to create your inventory file"
     Please visit this [dedicated section](../usage-inventory-catalog.md) for how to use inventory and catalog files.
 
-To run an EOS commands list on the reachable devices from the inventory:
+##### Run EOS commands
+
+> This script runs a list of EOS commands on reachable devices
+
```

Comment:
hehe we should probably write a test that is the script below to make sure we never break the doc.. maybe another PR!

Reply:
Surely another PR ;)

Reply:
yeah I opened an issue for it